src/Controller/DebugPagesController.php

```diff
@@ -0,0 +1,54 @@
+<?php declare(strict_types=1);
+/**
+ * @copyright Zicht Online <https://zicht.nl>
+ */
+
+namespace Zicht\Bundle\PageBundle\Controller;
+
+use Doctrine\ORM\EntityRepository;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Zicht\Bundle\PageBundle\Model\ContentItemInterface;
+use Zicht\Bundle\PageBundle\Model\PageInterface;
+
+class DebugPagesController extends AbstractController
+{
+    /**
+     * @Route("debug/pages")
+     */
+    public function showProjectPageTypeLinksAndInfo(): Response
+    {
+        $pageManager = $this->getPageManager();
+        /** @var EntityRepository $repository */
+        $repository = $pageManager->getBaseRepository();
+
+        $pagesInfo = [];
+        foreach ($pageManager->getPageTypes() as $pageType) {
+            $qb = $repository->createQueryBuilder('p');
+            $qb->where($qb->expr()->isInstanceOf('p', $pageType));
+            $qb->orderBy('RAND()');
+
+            $count = (clone $qb)->select('COUNT(p.id)')->getQuery()->getSingleScalarResult();
+            /** @var PageInterface $page */
+            $page = $qb->setMaxResults(1)->getQuery()->getOneOrNullResult();
+            $contentItemTypes = array_count_values(array_map(
+                function (ContentItemInterface $contentItem) {
+                    return $contentItem->getType();
+                },
+                $page ? $page->getContentItems() : []
```

Comment:
Ter info: in een ander project kreeg ik de error `Warning: array_map(): Expected parameter 2 to be an array, object given`. Ik heb deze regel toen vervangen door `$page ? iterator_to_array($page->getContentItems()) : []`.

Reply:
Aha. Misschien dan ook nog even een check of het wel een iterator (`\Traversable`?) is

Reply:
Deze heb ik ook ja